File: MCP服务器/finance/QuantConnect_mcp-server_3a656b0_python/tests/algorithms/charts.py
# region imports
from AlgorithmImports import *
# endregion


class ChartTestAlgorithm(QCAlgorithm):

    def initialize(self):
        self.set_start_date(2023, 1, 1)
        self.set_end_date(2023, 4, 1)
        symbol = self.add_equity("SPY", Resolution.DAILY).symbol
        self.plot_indicator("SMA", self.sma(symbol, 10))

File: MCP服务器/finance/QuantConnect_mcp-server_3a656b0_python/tests/algorithms/Charts.cs
#region imports
    using QuantConnect;
    using QuantConnect.Algorithm;
#endregion
public class ChartTestAlgorithm : QCAlgorithm
{
    public override void Initialize()
    {
        SetStartDate(2023, 1, 1);
        SetEndDate(2023, 4, 1);
        var symbol = AddEquity("SPY", Resolution.Daily).Symbol;
        PlotIndicator("SMA", SMA(symbol, 10));
    }
}
